nf_core/lint/__init__.py

```diff
@@ -470,7 +470,7 @@ def _get_results_md(self):
             test_failures = "### :x: Test failures:\n\n{}\n\n".format(
                 "\n".join(
                     [
-                        f"* [{eid}](https://nf-co.re/tools-docs/lint_tests/{eid}.html) - "
+                        f"* [{eid}](https://nf-co.re/tools/docs/{tools_version}/pipeline_lint_tests/{eid}.html) - "
```

Comment:
Just saw in the tests that `tools_version` is not defined, we should define it here as in [`format_result()`](https://github.com/nf-core/tools/blob/master/nf_core/lint/__init__.py#L365-L376)